### PR TITLE
Update KE versions

### PR DESCRIPTION
**Pull request by semyonmor**

### CODE REVIEW

File: kube-enforcer/values.yaml [hunk 1]
```diff
@@ -85,7 +85,7 @@ microEnforcerImage:
 # KubeBench Image
 kubebenchImage:
   repository: "aquasec/kube-bench"  # Default aqua registry KubeBench repository name
-  tag: "v0.6.15"
+  tag: "v.0.7.0"
```
Comment (deven0t) [inline kube-enforcer/values.yaml:88]: tag is having extra dot[.], it should be this `v0.7.0`

File: kube-enforcer/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ All notable changes to this project will be documented in this file.
 
 ## 2022.4.37 ( Jan 3rd, 2024 )
 * Updated enforcer chart version to 2022.4.19
+* Update startboard version to 0.15.9
+* Update kube-bench version to v.0.7.0
```
Comment (deven0t) [inline kube-enforcer/CHANGELOG.md:7]: same here, v0.7.0

File: kube-enforcer/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,8 @@ All notable changes to this project will be documented in this file.
 
 ## 2022.4.37 ( Jan 3rd, 2024 )
 * Updated enforcer chart version to 2022.4.19
+* Update startboard version to 0.15.9
```
Comment (deven0t) [inline kube-enforcer/CHANGELOG.md:6]: starboard version 0.15.19

**Review by deven0t — COMMENTED** [review]
Small comments

File: kube-enforcer/values.yaml (hunk 1 above)
Comment (semyonmor) [reply kube-enforcer/values.yaml]: fixed

File: kube-enforcer/CHANGELOG.md (hunk 2 above)
Comment (semyonmor) [reply kube-enforcer/CHANGELOG.md]: fixed

File: kube-enforcer/CHANGELOG.md (hunk 3 above)
Comment (semyonmor) [reply kube-enforcer/CHANGELOG.md]: fixed

**Review by deven0t — APPROVED** [review]
LGTM

**Pull request merged**